### ISSUE TITLE
[Dropdown] Clearable and multiple selection delete icon should be keyboard accessible

### ISSUE THREAD

**Issue by funelie** [fomantic/Fomantic-UI#3203]
# Bug Report
In clearable and mutliple selection fields, the "delete" icons on the clearable field and each "delete" icon on selected items are not keyboard focusable, and therefore not usable in keyboard only navigation.
The delete icon should also have an accessible name that states its function, for screen reader users, when focused.

## Steps to reproduce
1. Navigate on a multiple selection field using the keyboard only.
2. Focus on items using the arrow keys : the delete icon is never on focus and can't be focused.

## Expected result
When navigating the selected items, the delete icon for each item should be focusable.

## Actual result
The label of each selected item can be focused, but never the delete button within.

## Version

2.9.4
